src/jump_moi_overloads.jl

```diff
@@ -80,6 +80,42 @@ function MOI.get(
     return _moi_get_result(JuMP.backend(model), attr, JuMP.index(var_ref))
 end
 
+# extras to handle model_dirty
+
+function MOI.get(
+    model::JuMP.Model,
+    attr::ReverseConstraintSet,
+    var_ref::JuMP.ConstraintRef,
+)
+    JuMP.check_belongs_to_model(var_ref, model)
```

Comment:
What's the issue with `mode_dirty` for getters ? We should refer to a comment at the top of the file explaining it

Reply:
moved to the top